src/test/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/InvalidMultipolygonSlicingTest.java

```diff
@@ -224,14 +217,15 @@ public void testSelfIntersectingOuterRelationAcrossBoundary()
         Assert.assertEquals(10, rawAtlas.numberOfPoints());
         Assert.assertEquals(1, rawAtlas.numberOfRelations());
 
-        final Atlas slicedAtlas = RAW_ATLAS_SLICER.slice(rawAtlas);
+        final Atlas slicedAtlas = rawAtlasSlicer.slice(rawAtlas);
+        System.out.println(slicedAtlas);
 
         // Assert that we CAN build a valid building with this relation
         new ComplexBuildingFinder().find(slicedAtlas)
                 .forEach(building -> Assert.assertFalse(building.getError().isPresent()));
 
         Assert.assertEquals(29, slicedAtlas.numberOfPoints());
-        Assert.assertEquals(8, slicedAtlas.numberOfLines());
+        Assert.assertEquals(7, slicedAtlas.numberOfLines());
```

Comment:
Is that one of the Edge (case) lines?

Reply:
ahh, whoops, should have mentioned this; I'll update the PR with a description of what this unit test looks like with the new behavior

Reply:
one thing that's important to note here is that this *does* join the beginning and the end of the Line. I think this makes sense, because we're already splitting the Line into pieces, but *if* it's important to preserve that, we can easily disable the merge code and/or disallow merge for start and end pieces.